libs/header/p2p/metrics.go

```diff
@@ -0,0 +1,57 @@
+package p2p
+
+import (
+	"context"
+
+	"go.opentelemetry.io/otel/metric/global"
+	"go.opentelemetry.io/otel/metric/instrument"
+	"go.opentelemetry.io/otel/metric/instrument/syncfloat64"
+)
+
+type metrics struct {
+	responseSize     syncfloat64.Histogram
+	responseDuration syncfloat64.Histogram
+}
+
+var (
+	meter = global.MeterProvider().Meter("header/p2p")
+)
+
+func (ex *Exchange[H]) RegisterMetrics() error {
+	responseSize, err := meter.
+		SyncFloat64().
+		Histogram(
+			"header_p2p_headers_response_size",
```

Comment:
no need to copy header_p2p, as it's already in the meter name

Reply:
Same for others

Reply:
When the metrics are displayed, the meter name is not considered.

Reply:
It consireded in traces 100%, are you sure it's not there for metrics? I really don't like redundant/duplicate info, but that’s nit. The real concern is to keep this consistent in the node which is not at this point.